docs/posts/announcing-powershell-generator.md

```diff
@@ -0,0 +1,84 @@
+
+Today, we're announcing beta support for PowerShell in [AutoRest](https://aka.ms/autorest), so that you can now generate PowerShell modules from Swagger/OpenAPI JSON documents.
+
+> [AutoRest](http://github.com/Azure/autorest) is the SDK generation tool that we use in Azure to produce SDKS for 90+ management services across 7+ languages. <br>Its pluggable architecture allows fine-grained control over the generation process, and allows extensions to be written in any language that can read/write JSON via stdin/stdout (we use the [JSON-RPC](https://www.npmjs.com/package/vscode-jsonrpc) protocol that [Visual Studio Code uses](https://code.visualstudio.com) )
+
+Along the way, we had to go back and make some updates to the core of AutoRest (to begin support of OpenAPI 3, and introduce some changes to support generating multiple API versions with Azure Profiles.)
+
+
+## Getting Started
+
+### Requirements
+
+Use of the beta version of `autorest.powershell` requires the following:
+
+- [Node.js LTS](https://nodejs.org) (10.15.x LTS preferred. Will not function with a Node version less than 10.x. Be wary of 11.x builds as they may introduce instability or breaking changes. ) 
+> If you want an easy way to install and update Node, I recommend [NVS - Node Version Switcher](https://github.com/Azure/autorest/blob/master/docs/nodejs/installing-via-nvs.md) or [NVM - Node Version Manager](https://github.com/Azure/autorest/blob/master/docs/nodejs/installing-via-nvm.md)
+
+- [AutoRest](https://aka.ms/autorest) v3 beta: <br>`npm install -g autorest@beta`
+- PowerShell 6.1 - If you don't have it installed, you can use the cross-platform npm package <br> `npm install -g pwsh` <br>&nbsp;
+- Dotnet SDK 2 or greater - If you don't have it installed, you can use the cross-platform .NET 2.1 npm package <br> `npm install -g dotnet-sdk-2.1 ` <br>&nbsp;
+
+### Using AutoRest Powershell
+
+At a bare minimum, you can generate a PowerShell module using a Swagger or OpenAPI file and using `--powershell`.
+
+The output will be in the `./generated` folder by default:
+
+`autorest --powershell --input-file:<path-to-swagger-file> [...options]`
+
+Be sure to check out [these additional samples that use the PowerShell generator](https://github.com/Azure/autorest/blob/master/docs/powershell/samples/readme.md).
+
+### Known Issues
+As with all `beta` software, there are bound to be a few glitches or things that are not working. 
+
+We've cataloged some [known issues](https://github.com/Azure/autorest/blob/master/docs/powershell/release-notes.md#caveats-and-known-issues) with this first beta we encourage you to read before reporting any issues you experience.
+
+### Support 
+We're working as fast as we can to finish up the generator, as we have a lot of modules to generate internally. 
+
+If you run into problems, you can post an issue on the [github repo](https://github.com/Azure/autorest/issues) and tag it with the `powershell` label, and we'll try to take a look.
+
+## Features
```

Comment:
I would recommend putting **Features** before **Known Issues** and **Support** section.

Reply:
Uh, it was ... in my head 🌝 

.. fixed